src/search/components/JsonData/style.scss

```diff
@@ -0,0 +1,7 @@
+.JsonData .JsonData-json {
+  display: none;
+}
+
+.JsonData.JsonData-visible .JsonData-json {
```

Comment:
Personally I'm not sure I like the repitition of the namespace, it seems OTT. Could you not use:

```css
.JsonData.visible {}
.JsonData .json {}
```

I feel that would cover these cases just fine.

Either way I think it's important we find a consensus re: the approach to CSS so we have consistency across addons-frontend so we don't have a mix of different approaches.